source/sass/views/home.scss

```diff
@@ -89,6 +89,10 @@
       font-weight: bold;
     }
 
+    .card {
+      flex: 1;
```

Comment:
this should also work as the bootstrap `flex-grow-1` rule, so we can try to replace the `align-items-stretch` with that

Reply:
brilliant!